_posts/elements/2015-04-01-colors.md

```diff
@@ -13,441 +13,693 @@ slug:         colors
 
 description:  Pattern Library color swatches
 
-info: These are the predefined colors used throughout our patterns and elements. Values here (noted in RGB format as well by the relative scale name) are used for backgrounds, text color, and decorative elements. Note - when using any values here, make sure that any text on a background passes WCAG AA guidelines for color contrast.
+info: These are the predefined colors used throughout our patterns and elements. Values here (noted in RGB, HEX, and the Sass variable) are used for backgrounds, text color, and decorative elements. Note - when using any values here, make sure that any text on a background passes WCAG AA guidelines for color contrast.
```

Comment:
NIT: mind changing this order to match the order we present them in within the UI (Sass Var, RGB, Hex)?